### PR TITLE
qt: Fix for linuxbrew

### LINKED ISSUE #232: qt: No such file or directory
```
cp -f -r /home/user/hbetts/.local/tmp/qt-udQfzz/qt-everywhere-opensource-src-4.8.6/mkspecs/wincewm65professional-msvc2008 /home/user/hbetts/.local/Cellar/qt/4.8.6/mkspecs/
Error: No such file or directory @ unlink_internal - /home/user/hbetts/.local/Cellar/qt/4.8.6/bin/pixeltool.app

> brew --config
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/linuxbrew.git
HEAD: f044cda6a9d19d9981ef72e5e78bfa14e09f8e2c
Last commit: 2 days ago
HOMEBREW_PREFIX: /home/user/hbetts/.local
HOMEBREW_CELLAR: /home/user/hbetts/.local/Cellar
CPU: 24-core 64-bit 6
OS X: 0-x86_64
Xcode: N/A
CLT: N/A
Clang: N/A
X11: N/A
System Ruby: 2.2.0 => /home/user/hbetts/.local/var/rbenv/versions/2.2.0/bin/ruby
Perl: [5.12]
Python: /home/user/hbetts/.local/bin/python => /home/user/hbetts/.local/Cellar/python/2.7.9/bin/python2.7
Ruby: /home/user/hbetts/.local/var/rbenv/versions/2.2.0/bin/ruby
```

### PR DESCRIPTION
**Pull request by saketkc**
Fixes #232

### CODE REVIEW

**Comment by saketkc** [comment]
Possibly failing because of old gcc. Builds fine locally.

**Comment by sjackman** [comment]
You could test locally with `brew install qt --gcc=gcc-4.6` if you were inclined. Definitely a compile bug though.

```
../3rdparty/javascriptcore/JavaScriptCore/runtime/CallData.cpp
{standard input}: Assembler messages:
{standard input}:4609: Warning: end of file not at end of a line; newline inserted
{standard input}:5097: Error: bad register name `%rc'
{standard input}: Error: open CFI at the end of file; missing .cfi_endproc directive
g++-4.6: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-4.6/README.Bugs> for instructions.
```

File: Library/Formula/qt.rb [hunk 1]
```diff
@@ -91,9 +91,6 @@ def install
     ENV.j1
     system "make install"
 
-    # what are these anyway?
-    (bin+'pixeltool.app').rmtree
-    (bin+'qhelpconverter.app').rmtree
```
Comment (sjackman) [inline Library/Formula/qt.rb]: Please add `if OS.mac?` to both.

**Comment by sjackman** [comment]
Thanks for the fix, Saket.

File: Library/Formula/qt.rb (hunk 1 above)
Comment (saketkc) [inline Library/Formula/qt.rb]: Oops, yes!

**Pull request closed without merging**